frontend/packages/jobs/src/prompts/generateReview/reviewSchema.ts

```diff
@@ -23,26 +14,26 @@ const SeverityEnum = enumType({
   POSITIVE: 'POSITIVE',
 })
 
-export const reviewSchema = object({
+export const reviewSchema = strictObject({
   bodyMarkdown: string(),
   issues: array(
-    object({
+    strictObject({
       kind: KindEnum,
       severity: SeverityEnum,
       description: string(),
       suggestion: string(),
       suggestionSnippets: array(
-        object({
+        strictObject({
           filename: string(),
           snippet: string(),
         }),
       ),
     }),
   ),
   scores: array(
-    object({
+    strictObject({
       kind: KindEnum,
-      value: pipe(number(), minValue(0), maxValue(10)),
+      value: number(),
```

Comment:
Using the `minimum` property seems to trigger an error. I'll look into it at a later time. 🤔 


```
{"error":{"message":"Invalid schema for response_format 'review': In context=('properties', 'scores', 'items', 'properties', 'value'), 'minimum' is not permitted.","type":"invalid_request_error","param":"response_format","code":null}}
```